integration/commitlog_bootstrap_merge_test.go

```diff
@@ -56,6 +56,10 @@ import (
 // - ns1 block at t0 and t1
 // - commit log blocks from [t1, t3]
 func TestCommitLogAndFSMergeBootstrap(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
```

Comment:
Hm I wish there was a way to enforce this.

Reply:
Either way, can open that up as a separate task.

Reply:
lol tried migrating to that but aggregator uses go1.8 for ci tests still :| there's a TODO in the ci-scripts module to address.